### PR TITLE
[refactor][alert] refactor alert module

### PR DESCRIPTION
**Pull request by Zzm0809**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Comment by leechor** [comment]
LGTM

File: dinky-alert/dinky-alert-base/src/main/java/org/dinky/alert/AlertTypeEnum.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+public enum AlertTypeEnum {
+    /** DingTalk */
+    DINGTALK("DingTalk"),
+    /** FeiShu */
+    FEISHU("FeiShu"),
+    /** Email */
+    EMAIL("Email"),
+    /** FeiShu */
+    WECHAT("FeiShu");
```
Comment (zackyoungh) [inline dinky-alert/dinky-alert-base/src/main/java/org/dinky/alert/AlertTypeEnum.java]: 名字错了

**Review by zackyoungh — APPROVED** [review]
LGTM

File: dinky-alert/dinky-alert-base/src/main/java/org/dinky/alert/AlertTypeEnum.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.dinky.alert;
+
+public enum AlertTypeEnum {
```
Comment (leechor) [inline dinky-alert/dinky-alert-base/src/main/java/org/dinky/alert/AlertTypeEnum.java]: AlertType without Enum is better.

File: dinky-alert/dinky-alert-base/src/main/java/org/dinky/alert/ShowType.java [hunk 3]
```diff
@@ -26,12 +26,18 @@
...
-    ATTACHMENT(3, "attachment"), // 邮件相关 只发送附件
-    TABLE_ATTACHMENT(4, "table attachment"); // 邮件相关 邮件表格+附件
+    /** markdown */
+    MARKDOWN(0, "markdown"),
+    /** text */
+    TEXT(1, "text"),
+    /** post of feishu */
+    POST(2, "post"),
+    /** table */
+    TABLE(0, "table"),
```
Comment (leechor) [inline dinky-alert/dinky-alert-base/src/main/java/org/dinky/alert/ShowType.java]: same code as Markdown?

**Review by leechor — APPROVED** [review]

**Review by aiwenmo — CHANGES_REQUESTED** [review]
Please remove AlertTypeEnum

**Review by aiwenmo — APPROVED** [review]

**Pull request merged**